crates/rustc_codegen_spirv/src/linker/test.rs

```diff
@@ -77,7 +76,6 @@ fn assemble_and_link(binaries: &[&[u8]]) -> Result<Module, String> {
         output_file: None,
         output_dir: None,
         file_loader: None,
-        diagnostic_output: DiagnosticOutput::Raw(Box::new(write_diags)),
```

Comment:
This breaks the tests, we lost an override and have to rewrite this system, which will block your PR on some of the cleanups I've been doing on the SPIR-T integration side.

Reply:
I'm particularly having issues with this. After removing it, RA says it's missing that field (go figure). When I go the definition, the field is in there as well. So I don't understand the compile error in the first place.

Reply:
I'll go back to 2022-10-15 then until this is fixed